### PR TITLE
Support enums in properties

### PR DESCRIPTION
**Pull request by leolplex**
Hello @StefH ,

The idea here is support that when a property will have an enum one of them is selected random.

### CODE REVIEW

**Comment by StefH** [comment]
@leolplex 
I did just merge your previous PR to master, so now you need to merge/rebase and fix merge conflicts.

**Review by StefH — APPROVED** [review]

File: src/WireMock.Net.OpenApiParser/Utils/ExampleValueGenerator.cs [hunk 1]
```diff
@@ -38,49 +41,78 @@ public object GetExampleValue(OpenApiSchema schema)
...
                             var exampleString = (OpenApiString)schemaExample;
-                            return exampleString?.Value ?? _settings.ExampleValues.String;
+                            var enumString = (OpenApiString)schemaEnum;
+                            var valueStringEnumOrExample = enumString is null ? exampleString?.Value : enumString?.Value;
+                            return valueStringEnumOrExample ?? _settings.ExampleValues.String;
                     }
             }
         }
+
+        private IOpenApiAny GetEnum(IList<IOpenApiAny> schemaEnum)
```
Comment (StefH) [inline src/WireMock.Net.OpenApiParser/Utils/ExampleValueGenerator.cs:107]: `private static IOpenApiAny GetRandomEnumValue(IList<IOpenApiAny> schemaEnum)`

Comment (leolplex) [reply src/WireMock.Net.OpenApiParser/Utils/ExampleValueGenerator.cs]: @StefH Done!

**Review by StefH — APPROVED** [review]

**Comment by leolplex** [comment]
Hello @StefH, could you merge this PR?

Any chance to deploy a new nuget version with mi last 3 PR (#681 #678 #680), please?  Thanks 

Daniel.

**Pull request merged**